flink-connectors/flink-connector-kinesis/src/test/java/org/apache/flink/streaming/connectors/kinesis/FlinkKinesisConsumerTest.java

```diff
@@ -216,14 +217,73 @@ public void testUnixTimestampInValidateOptionDatePropertyForInitialTimestampInCo
 		testConfig.setProperty(ConsumerConfigConstants.STREAM_INITIAL_POSITION, "AT_TIMESTAMP");
 		testConfig.setProperty(ConsumerConfigConstants.STREAM_INITIAL_TIMESTAMP, unixTimestamp);
 
-		KinesisConfigUtil.validateConsumerConfiguration(testConfig);
+		try {
+			KinesisConfigUtil.validateConsumerConfiguration(testConfig);
 
-		try{
 			double value = Double.parseDouble(unixTimestamp);
 			if (value < 0) {
 				throw new NumberFormatException();
 			}
```

Comment:
Because I want to force the value be positive. Is it okey to let it be negative?